data/jsons/samples_blacklist.json

```diff
@@ -0,0 +1 @@
+["http-composed.fbp"]
```

Comment:
maybe you could name it as samples_dependency_check_skip_list or anything more descriptive than just a blacklist file on our data dir.